examples/benchmark-memory.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+declare(strict_types=1);
```

Comment:
Lose the blank line before.

```suggestion
declare(strict_types=1);
```